components/deployment-automation/org.wso2.carbon.deployment.automation/src/main/java/org/wso2/carbon/deployment/automation/DeploymentService.java

```diff
@@ -0,0 +1,194 @@
+/*
+ * Copyright (c) 2016, WSO2 Inc. (http://wso2.com) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.wso2.carbon.deployment.automation;
+
+import io.swagger.annotations.*;
```

Comment:
It would be better to use specific imports without using asterisk. This will get validated via checkstyle if we add wso2 parent pom.xml. Thanks!